RELEASE.md

```diff
@@ -0,0 +1,84 @@
+
+* Install [git-extras](https://github.com/visionmedia/git-extras) and run `git changelog` to update `History.md`. Clean up the changelog, removing merge commits, whitespace cleanups or other irrelevant commits. Refer to the formatting of previous entries when in doubt.
+* Run `grunt authors` and manually add any new authors to `AUTHORS.txt`. The order of this list doesn't change, so you should compare starting at the end.
+* Update the version property in `package.json` to have the right `-pre` version (e.g., if releasing version `2.3.0`, the version should be `2.3.0-pre`). This is not necessary for patch releases.
+
+Commit all of the above with the following message:
+
+	Build: Prepare @VERSION release, including authors and history update
+
+Replace `@VERSION` with the version number you are releasing.
+
+For the final preparatory step, push your new commit to `master` on GitHub. In order to do this, you'll need to disable branch protections for administrators. This can be done [here](https://github.com/qunitjs/qunit/settings/branches/master).
+
+## Performing the Release
+
+For the actual release, we will use [QUnit's fork of jquery-release](https://github.com/qunitjs/jquery-release).
+
+:warning: **WARNING** :warning:
+> Before attempting the process below, make sure you have the proper permissions to publish to the jQuery CDN or else you will run into issues. If you have any doubts, ask someone who has done a release before to verify for you.
+
+Always start this process from a fresh clone of the release repo:
+
+	git clone https://github.com/qunitjs/jquery-release.git
+	cd jquery-release
+
+Then simply run the script:
+
+	node release.js --remote=qunitjs/qunit
+
+Follow the prompts along the way and you should be all good. This process will create a new release and tag, push it to GitHub, and publish it to NPM, Bower, and the jQuery CDN.
+
+You can verify all of the above were correctly published by using the following:
+
+- Bower: `bower info qunit`
+- NPM: `npm view qunitjs`
+- CDN: visit `https://code.jquery.com/qunit/qunit-x.x.x.js`
+
+## Updating the Website
+
+Once you have successfully published the new release, we need to update the website.
+
+As with QUnit itself, checkout the current `master` version of the website repo, ensuring that any necessary updates for this release have made it into the code.
+
+Next, do a find and replace of the previous version number and insert the new version number (excluding `package.json`, it will be updated separately). Commit those changes with the following message:
+
+	All: Update url and version to @VERSION
+
+Next, change the version in `package.json` to the next logical number. Usually, this will be the same as the `@VERSION`, but occassionally the website will be a patch version ahead if a bug occurred in the website itself. Commit that change with the following message:
+
+	@WEBSITE_VERSION
+
+Finally, tag the version and push it and the changes to `master`:
+
+	git tag "vX.X.X"
```

Comment:
for the commit and tag I usually use `npm version @VERSION` which will update the package.json, create the commit and tag. It's basically a shortcut for these steps.